rust/numaflow-core/src/shared/server_info.rs

```diff
@@ -18,6 +18,17 @@ use crate::shared::server_info::version::SdkConstraints;
 // Equivalent to U+005C__END__.
 const END: &str = "U+005C__END__";
 
+pub const CONTAINER_TYPE_SOURCER: &str = "sourcer";
```

Comment:
What is the best way to do this in rust?

Reply:
Enum?

Reply:
Enum with string? Not seems to work.

Reply:
Do you want something like this?
https://github.com/numaproj/numaflow/pull/2020

Reply:
Constant seems to be simpler for this specific use case.

Reply:
idiomatic rust is to use enum and `impl std::fmt::Display` for getting the string version string.